sources/engine/Xenko.Voxels/Xenko.Voxels.csproj

```diff
@@ -0,0 +1,715 @@
+﻿<Project>
+  <PropertyGroup>
+    <XenkoAssemblySign>true</XenkoAssemblySign>
+    <XenkoRuntime>true</XenkoRuntime>
+    <XenkoRuntimeNetStandard>true</XenkoRuntimeNetStandard>
+  </PropertyGroup>
+  <Import Project="..\..\targets\Xenko.PreSettings.targets" />
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+  <PropertyGroup>
+    <XenkoAssemblyProcessor>true</XenkoAssemblyProcessor>
+    <XenkoPlatformDependent>true</XenkoPlatformDependent>
+    <XenkoBuildTags>*</XenkoBuildTags>
+    <XenkoPackAssets>true</XenkoPackAssets>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="..\..\shared\SharedAssemblyInfo.cs">
+      <Link>Properties\SharedAssemblyInfo.cs</Link>
+    </Compile>
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\Xenko.Engine\Xenko.Engine.csproj" />
+    <ProjectReference Include="..\Xenko.Native\Xenko.Native.csproj" />
+    <ProjectReference Include="..\Xenko.Games\Xenko.Games.csproj" />
+    <ProjectReference Include="..\Xenko.Rendering\Xenko.Rendering.csproj" />
+  </ItemGroup>
+  <ItemGroup>
+    <Service Include="{508349B6-6B84-4DF5-91F0-309BEEBAD82D}" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Update="Voxels\GraphicsCompositor\DebugVisualizations\Shaders\VoxelVisualizationRawEffect.cs">
```

Comment:
All of those `Compile` shouldn't be necessary with latest `master` (xksl/cs files are automatically included/setup).
Let me know if you need help cleaning it, but basically removing the two `ItemGroup` with all those `Compile` and `None` items should do it.

Reply:
Will update to the latest master and fix it up, thanks :)

Reply:
Thanks! And if you have any trouble with it, just let me know, I can do it on my side!